packages/manager/apps/pci-load-balancer/src/components/create/InstanceTable.component.tsx

```diff
@@ -116,9 +116,9 @@ export function InstanceTable({
               ],
             },
           ]);
-          tracking?.trackClick({
-            name: `${TRACKING_PREFIX}::add-listener`,
-            type: 'action',
+          trackClick({
+            actions: [`${TRACKING_PREFIX}`, 'add-listener'],
```

Comment:
I don't think TRACKING_PREFIX should be kept when using useOvhTracking, can you verify ?